docs/samples/browser-plugin-meetings/index.html

```diff
@@ -1,508 +1,594 @@
-            <li><a href="#meetingControlsScreenshare">Screenshare</a></li>
-            <li><a href="#meetingControlsRecording">Recording</a></li>
-            <li><a href="#meetingControlsLocking">Lock / Unlock</a></li>
-            <li><a href="#meetingControlsTranscription">Transcription</a></li>
-            <li><a href="#meetingControlsTones">Tones</a></li>
-          </ol>
-          <br>
-
-          <!-- meeting-controls / mute-unmute audio -->
-          <fieldset id="meetingControlsMuteAudio">
-            <legend>Mute/Unmute Audio</legend>
+      <!-- meeting streams -->
+      <section class="section-box">
+        <h2 class="collapsible">Meeting Streams</h2>
+        <div class="section-content">
+          <div class="video-section">
             <div>
-                <p class="note">NOTE: Must have an active meeting object.</p>
-                <div class="u-mb">
-                  <button id="ts-toggle-audio" type="button" onclick="toggleSendAudio()">Toggle Audio</button>
-                  <div class="btn-group">
-                      <button type="button" class="btn-code" onclick="meeting.muteAudio()">meeting.muteAudio()</button>
-                      <button type="button" class="btn-code" onclick="meeting.unmuteAudio()">meeting.unmuteAudio()</button>
+              <div class="flex" style="gap: 0.5rem;">
+                <fieldset>
+                  <legend>Local Video</legend>
+                  <video id="local-video" muted="true" autoplay playsinline style="height: 200px;"></video>
+                </fieldset>
+
+                <fieldset style="display: flex; align-items: center;">
+                  <legend>Local Audio</legend>
+                  <audio id="local-audio" controls autoplay></audio>
+                </fieldset>
+              </div>
+              <div class="flex">
+                <!--mute-unmute audio -->
+                <div>
+                  <div class="u-mb">
+                    <button id="ts-toggle-audio" type="button" onclick="toggleSendAudio()">Toggle Audio</button>
                   </div>
                 </div>
-                <pre id="ts-toggle-audio-status"></pre>
-            </div>
-          </fieldset>
 
-          <!-- meeting-controls / mute-unmute-video -->
-          <fieldset id="meetingControlsMuteVideo">
-            <legend>Mute/Unmute Video</legend>
-            <div>
-                <p class="note">NOTE: Must have an active meeting object.</p>
-                <div class="btn-group">
-                  <button type="button" class="btn-code" onclick="meeting.muteVideo()">meeting.muteVideo()</button>
-                  <button type="button" class="btn-code" onclick="meeting.unmuteVideo()">meeting.unmuteVideo()</button>
+                <!--mute-unmute-video -->
+                <div>
+                  <div class="u-mb">
+                    <button id="ts-toggle-video" type="button" onclick="toggleSendVideo()">Toggle Video</button>
```

Comment:
Same here, I feel, Simply we can say Mute Video or Unmute Video. Or best will be to use the Icons for the best UI.